### PR TITLE
fix: add sp exit status check when craete gvg

### PR DESCRIPTION
**Pull request by Fynnss**
### Description

Add sp exit status check when craete gvg

### Rationale

Add sp exit status check when craete gvg

### Example

NA

### Changes

Notable changes: 
* NA

### CODE REVIEW

**Review by will-2012 — APPROVED** [review]

File: x/virtualgroup/keeper/msg_server.go [hunk 1]
```diff
@@ -56,6 +56,10 @@ func (k msgServer) CreateGlobalVirtualGroup(goCtx context.Context, req *types.Ms
 		return nil, sptypes.ErrStorageProviderNotFound.Wrapf("The address must be operator address of sp.")
 	}
 
+	if !sp.IsInService() {
```
Comment (alexgao001) [inline x/virtualgroup/keeper/msg_server.go:59]: should also allow Maintenance SP otherwise it is not able to conduct self test if no GVG created yet?

Comment (Fynnss) [reply x/virtualgroup/keeper/msg_server.go]: Fixed. Does sp in maintenance can send swapout tx?

Comment (alexgao001) [reply x/virtualgroup/keeper/msg_server.go]: it is not allowed.

Comment (Fynnss) [reply x/virtualgroup/keeper/msg_server.go]: yes, make sense. Please review this pr again. thx.

**Review by alexgao001 — APPROVED** [review]

**Pull request merged**